installer/packer/scripts/kov/configure_kov.sh

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/bash
```

Comment:
why does all the cert generation happen on every restart of the service, shouldn't this happen in the provision script?
It might be better to run this script once and make it produce an env file in /etc/default/kovd which we can later source with the EnvironmentFile entry in the kov.service script